README.md

```diff
@@ -392,6 +392,23 @@ api.claimAssets(
 );
 ```
 
+### Estimating XCM Fees on Destination Chain
+
+In combination with the XCM Dry Run API, The api can be used to estimate fees for the xcm's forwarded to other chains.
```

Comment:
```suggestion
In combination with the XCM Dry Run API, this API can be used to estimate fees for XCM messages sent to other chains.
```
Just a suggestion. No need to change it!

Reply:
Thanks! Updated this since it reads better that way :)